website/docs/getting-started/2022-08-05-getting-started.md

```diff
@@ -0,0 +1,93 @@
+keywords: [podman desktop, podman, containers]
+hide_table_of_contents: false
+---
+
+<!--truncate-->
+
+## Introduction
+
+Podman Desktop leverages [Podman Engine](https://podman.io/) to provide a lightweight and daemon-less container management tool. It enables you to easily work with containers from your local environment without running container management commands. 
+
+Podman Desktop brings all the capabilities of Podman with the added advantage of a GUI which is becoming increasingly powerful and user-friendly. It is super easy to adapt to. With this tutorial, you will be ready to roll and work with your containerized applications in no time.
+
+In this tutorial, I am working with Podman Desktop v0.0.5.
+
+## Requirements
+
+In order to work with Podman Desktop, you firstly require the latest version of Podman Desktop. Depending on your Operating System, you can download it from [this](https://podman-desktop.io/downloads) page.
+
+Next, you need the latest version of Podman Engine installed on your computer. When started, Podman Desktop looks for the Podman Engine in the system and will prompt you to install the same if it is not present. You can read more about how to set up the Podman Desktop and Podman Engine in the Installation guide [here](/docs/installation/macos-install).
+
+Once everything is set up, you are ready to go!
+
+## Get Started
+
+When you open the Podman Desktop application, you will be greeted with the home screen. It will show you a toggle button that says ‘Initialize Podman’. And as mentioned, it initializes the Podman Engine and creates a Podman Machine for you. Click on it to create a Podman Machine. Within no time, your Podman Machine will be created. This is equivalent to running the command ```podman machine init```. You can read more about it [here](https://docs.podman.io/en/latest/markdown/podman-machine-init.1.html).
+
+Next, you need to start the Podman Machine by toggling the button that says ‘Run Podman’. This is equivalent to running ```podman machine start```. You can read more about it [here](https://docs.podman.io/en/latest/markdown/podman-machine-start.1.html). 
+
+Once it says “Podman is running”, you are ready to build images and run containers! 
+
+Our first step will be to build an image for our application followed by running the image to create the container.
+
+### 1. Build Image
+
+For this section of the tutorial, I have created my Containerfile using the Red Hat Universal Base Image. If you are following along and want to try it yourself, you can find the Containerfiles in [this repository](https://github.com/deekay2310/containerfile-playground). You can also write your own Containerfiles and build images from it using the Red Hat Universal Base Image (UBI). Read more about it in [this blog](https://developers.redhat.com/articles/2021/11/30/build-lightweight-and-secure-container-images-using-rhel-ubi). Once you have your Containerfile ready, follow the steps below to build your own image.
```

Comment:
If it's part of the Doc probably it needs to be into containers/ org and not on user namespace if we need to do changes

Reply:
Since the content is from the then planned blog post authored by me, the file(s) were uploaded in my repo. It can be kept there for now if you want. Or let me know where exactly you would want me to upload the file?